frontend/pages/wiki/_page.vue

```diff
@@ -17,7 +16,7 @@ export default {
   },
 
   created () {
-    this.pageName = this.removeLocalePrefix(this.$route.params.page)
```

Comment:
I think that all the changes from here to the end are related to try to fix broken links
If so, that was fixed in this commit: https://github.com/PhiloBiblon/philobiblon-ui/commit/a761338ca3fe2bbf7b05b773407877d361278f37
Let's discuss in our next meeting